app/src/org/commcare/utils/FormUploadUtil.java

```diff
@@ -56,7 +56,7 @@
 
     // This list is also maintained server side and any changes to this should be replicated on server as well
```

Comment:
I cannot find this list on HQ. Is this line still true?

Reply:
we actually didn't add it eventually. It was present on the PR but later reverted. Apologies, i should have notified you.

https://github.com/dimagi/commcare-hq/pull/28458#issuecomment-703786770
So eventually we just added check for the xml file extension.